src/main/java/com/snowflake/kafka/connector/internal/streaming/schemaevolution/iceberg/ApacheIcebergColumnSchema.java

```diff
@@ -0,0 +1,24 @@
+package com.snowflake.kafka.connector.internal.streaming.schemaevolution.iceberg;
+
+import org.apache.iceberg.types.Type;
+
+/** Wrapper class for Iceberg schema retrieved from channel. */
+public class ApacheIcebergColumnSchema {
+
+  private final Type schema;
+
+  private final String columnName;
```

Comment:
I need to have a column field. I can't retrieve it from schema.

Reply:
I don't understand this comment.  Could you elaborate on that?

Reply:
When `channel.getTableSchema().get("COLUMN_NAME").getIcebergSchema()` in return we receive schema like in my Tree test. There is no info about column name.
Hence I will have to get column name from InsertError. I am going to pass both into this class.

Reply:
ok, I got it